front/src/utils/types.ts

```diff
@@ -15,3 +15,5 @@ export type ArrayElement<ArrayType extends readonly unknown[] | undefined | null
 export type typedEntries<T> = Array<[keyof T, T[keyof T]]>;
 
 export type Unit = 'meters' | 'millimeters';
+
+export type Dictionary<T> = Record<string, T>;
```

Comment:
Hm, not a huge fan of type definitions like these:

- Dictionaries aren't allowed to have e.g. `number` as keys?
- It's small/readable enough to be inlined IMHO, creating an indirection isn't worth saving a few characters.

Reply:
Done